tests/Microsoft.Identity.Test.Unit/CoreTests/RegionDiscoveryProviderTests.cs

```diff
@@ -179,7 +179,6 @@ public async Task SuccessfulResponseFromRegionalizedAuthorityAsync()
         }
 
         [DataTestMethod]
-        [DataRow("Region with spaces")]
```

Comment:
Regions with spaces are no longer invalid as we account for them in RegionManager.cs

Reply:
nit: not much point for a having a DataTestMethod if there is only 1 data row ...